lib/internal/modules/esm/loader.js

```diff
@@ -14,14 +14,14 @@ const {
 
 const {
   ERR_INVALID_ARG_VALUE,
+  ERR_INVALID_ARG_TYPE,
```

Comment:
nit: move `ERR_INVALID_ARG_TYPE` above `ERR_INVALID_ARG_VALUE` to maintain alphabetical order.